src/tests/src/api/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+set(EXECUTABLE_NAME test-api)
+add_executable(${EXECUTABLE_NAME})
+
+target_sources(${EXECUTABLE_NAME}
+    PRIVATE
+    test_api.cpp
+        ${CMAKE_SOURCE_DIR}/api/API.cpp
+)
```

Comment:
Can't see any test_api.cpp in this PR, is this a mistake ?

Reply:
Maybe cherry-pick error